### PR TITLE
[Stream] Fixed bug in stream-pipeline for FA

### PR DESCRIPTION
**Pull request by sjw36**
* updated gemm tutorial for num_stages=0

### CODE REVIEW

File: python/tutorials/03-matrix-multiplication.py [hunk 1]
```diff
@@ -174,14 +174,18 @@
...
-        triton.Config({'BLOCK_SIZE_M': 128, 'BLOCK_SIZE_N': 256, 'BLOCK_SIZE_K': 64, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=4),
-        triton.Config({'BLOCK_SIZE_M': 64, 'BLOCK_SIZE_N': 256, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=4),
-        triton.Config({'BLOCK_SIZE_M': 128, 'BLOCK_SIZE_N': 128, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=4),
-        triton.Config({'BLOCK_SIZE_M': 128, 'BLOCK_SIZE_N': 64, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=4),
-        triton.Config({'BLOCK_SIZE_M': 64, 'BLOCK_SIZE_N': 128, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=4),
-        triton.Config({'BLOCK_SIZE_M': 128, 'BLOCK_SIZE_N': 32, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=4),
-        triton.Config({'BLOCK_SIZE_M': 64, 'BLOCK_SIZE_N': 32, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=2),
-        triton.Config({'BLOCK_SIZE_M': 32, 'BLOCK_SIZE_N': 64, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=2),
+        triton.Config({'BLOCK_SIZE_M': 128, 'BLOCK_SIZE_N': 256, 'BLOCK_SIZE_K': 64, 'GROUP_SIZE_M': 8}, num_stages=0, num_warps=4),
```
Comment (zhanglx13) [inline python/tutorials/03-matrix-multiplication.py:177]: Did you tune those gemm sizes to get these configs?

**Review by zhanglx13 — COMMENTED** [review]
@scxiao Are you tuning the gemm configs?

File: python/tutorials/03-matrix-multiplication.py (hunk 1 above)
Comment (sjw36) [reply python/tutorials/03-matrix-multiplication.py:177]: just changed to num_stages=0.. let me run tuning..

Comment (sjw36) [reply python/tutorials/03-matrix-multiplication.py]: updated configs

**Review by zhanglx13 — APPROVED** [review]

**Pull request merged**